### ISSUE TITLE
Questions about this

### ISSUE THREAD

**Issue by tedomedo** [tsgrgo/windows-update-disabler#1]
Can this work on win 10 home? If yes, then create such file for disabling Defender's update too.

**Comment by tsgrgo** [comment]
Yes, it does work. I tested it on my laptop running windows 10 home. I haven't thought about disabling defender updates yet but might look into it in the future.

**Comment by Cnhhc** [comment]
How about Windows10Pro for Workstations?
